src/services/mondomarket/index.jsx

```
@@ -264,6 +267,315 @@ export const AddMondoItem = ({ setAddModal, dispatch, setReload, regions }) => {
+        newFiles.map((file) => uploadImage(file))
+      );
+
+      const allImages = [
+        ...oldImages.map((img) => img.image),
+        ...uploadedNewImages,
+      ];
+
+      const payload = {
+        ...data,
+        paid_status: true,
+        oldImagesId: item?.images?.map((img) => img.id) || [],
```

ESLint

[error] 'item.images' is missing in props validation

(react/prop-types)